### DISCUSSION TITLE
2024-07-10_수_멘토링 1회차

### DISCUSSION THREAD

**Discussion by tttghost** [tttghost/wheelmaker#1] [General]
20240710_수_23:00~24:00 계획, 아이데이션

1. 계획
>월 수 금 1시간씩, 토or일 합 3시간 = 6시간 분배
월수금 못 채우는 날이 생기면 주말에 꼭 채우기
평일, 밤 11:00에 시작 1시간

2. 아이데이션
>주제는 xx키우기 로 생각
질병키우기, 버그키우기, 알키우기, 바람키우기...
특정 레벨업에 맞게 성장해야 하기 때문에 종류가 최소 10가지 이상의 것으로...
특이한 이름의 키우기도 좋을 것 같음, 저주키우기, 반죽키우기, 회전키우기(회전하는 물체 뭐든지간에 키우기, 바퀴,소용돌이,배수구구멍...), 바퀴키우기, 시간키우기, 수치심키우기, 갈고리 키우기, 
기본은 클리커 게임이면서 다른 장르와 합치면 좋을 것 같음

>갈고리키우기 좋을 것 같다. 회전하는 갈고리를 만든다음 놓았을 때  특정 영역에 잘 걸쳐지면 보상을 획득
보상을 획득하면 연속획득 보너스
기본적으로 빠른 클릭의 클리커게임 vs 타이밍을 맞춰 하는 리듬형식 게임

>바퀴키우기도 좋을것 같음
다양한 종류의 바퀴, 차바퀴, 책상바퀴, 벌레바퀴, 단추바퀴, 마차바퀴, 손익(소닉)바퀴, 캣휠바퀴, 챗바퀴, 공벌레바퀴, 표창바퀴, 선풍기날개바퀴, 소세지바퀴, 로딩바퀴, 오토바이바퀴, 티코바퀴, 동전바퀴, 지구바퀴, 도넛바퀴, (초코파이)정바퀴, 반바퀴(반잘려있는), 태극바퀴, 축구공바퀴, 네모바퀴, 너트바퀴(육각), 막대사탕바퀴, 관람차바퀴(실제 관람차의 액션도 구현하면 좋을듯), 수정구슬 바퀴, 눈결정바퀴, 물방울바퀴(멘트 - 공중에서 떨어지는 찰나의 순간을 바퀴로 재현하다! 물빠짐바퀴, 태풍바퀴, 가르마바퀴, 마법바퀴(닥터스트레인지), 바퀴순, 파괴왕바퀴(주호민일러스트이미지, ), 
기본 회전하는 시스템으로 재미있는 것이 나올 수 있을 것 같음...
계속 도는 바퀴 이미지를 사용해서 png재사용으로 이미지 쉽게 구성할 수 있을듯
각 바퀴에는 1줄 설명글이 달려있어도 좋을듯 (ex 네모바퀴 - 꼭 바퀴가 둥글란 법은 없자나?)
드래그를 해서 바퀴를 돌려서 속도를 올려 점수를 획득하는 시스템
바퀴에 눈 달아서 귀엽게 해도 될듯
아이디어 : 드래그해서 굴러다니는걸 튕기게 해서 재화획득을 하는방식으로
각 바퀴들을 레벨업을 통해 하나씩 얻는게 아니라 특정 보상으로 얻어서 도감채우는 형태로 가도 좋지않을까??
그럼 업데이트시 마다 도감채우기 용으로 바퀴를 모을 수 있을듯
각 바퀴마다 고유능력이나 고유이펙트 있으면 좋을듯

3. 재화
>재화는 1종류
단위는 원
돈 늘어나는 조건이 여러가지로
노멀겟, 배율1, 시간당 늘어나는 것
클릭겟, 배율2, 클릭으로 늘어나는 것
피버겟, 배율5, 피버타임때 클릭으로 늘어나는 것
피버타임은 최초 30분에 한번씩, 접속하지 않더라도 피버시간은 줄어듬, 클릭하면 더 빨리 줄어듬

3-1. 레벨업 시스템
>재화 획득 방식 증가
노멀겟 증가
클릭겟 증가
피버겟 증가

4. 이벤트시스템
>특정상황에 특수하게 발생되는 이벤트가 있으면 좋겠다.
발생된 이벤트에 따라 다량의 재화를획득할 수 있다. 
아니면 즉시 피버타임이 발동된다.

5. 볼거리
>특정 레벨구역이 있고 그 레벨 구역마다 볼거리가 있어야 한다. 올렸을 때 재미있을만한 보상,
다음 레벨업은 어떤 황당한 보상이 있을까? 를 상상하게 만들만한 재미있는 보상으로 구성해야 한다.

6. 도감시스템
>레벨업 해왔던 것들을 도감시스템으로 모으는 재미를 준다.

7. 장신구
>돈으로 바퀴의 눈코입 부분에 장신구를 달아준다.

**Comment by jongfeel** [comment]
## Feedbacks

저번에 얘기했던 거에서 조금 더 구체적으로 나왔군요.
이정도 명세가 나왔으면 프로젝트 세팅 하고 설계를 해봐도 좋을 것 같다는 생각입니다.

요일별 시간 할당이 잘 될 수 있도록 하고
또 몇 개의 task가 진행될 수 있게 해 보면 좋을 것 같습니다.

아래 할 일 목록에서 두 번째인 Github project는 제가 세팅하는 걸 도와드릴께요.

## To do

- [x] Create unity project repository and invite me
- [x] Set up Github project by projects
- [ ] Create tasks by issues

**Reply by tttghost** [reply]
네 레포지토리 생성 및 초대발송 하였습니다!

**Comment by jongfeel** [comment]
그리고 프로젝트는 끊임 없는 변화를 수용하고 일을 진행해 나가는 방식이 좋습니다.
즉, 설계, 구현이 진행되면서 기획 및 명세 내용도 변화가 있어야 한다는 걸 알고 계셔야 합니다.

**Reply by tttghost** [reply]
변화에 수용하는 것
네넵!
